docs/v2/2_installation.md

```diff
@@ -9,31 +9,27 @@ sidebar_label: Installation
 
 :::info
 
-In this section, we will go through the steps necessary to install the web3.unity SDK into the Unity3D game engine.
+In this section, we will go through the steps necessary for installing the web3.unity SDK into the Unity3D game engine. We will follow the installation with an example of how to implement a sample web3 login screen.
 
 :::
 
+## Installing web3.unity
+
 ### Install MetaMask
 
-You will need to have MetaMask installed in order to make web3-enabled games. If you already have MetaMask installed, you can skip this section and navigate to the [Downloading SDK](#downloading-the-sdk) section. If you have not installed a web3 wallet, please visit [MetaMask](https://t.co/8UNUrkShG6) for installation instructions.&#x20;
+You will need to have MetaMask & the appropriate browser extension installed in order to make web3-enabled games. If you already have MetaMask installed, you can skip this section and navigate to the [Downloading The SDK](#downloading-the-sdk) section. If you have not installed a web3 wallet, please visit [MetaMask](https://t.co/8UNUrkShG6) for installation instructions.&#x20;
```

Comment:
This reads weird bc now it makes me think that there is an additional extension needed beyond MM.

Reply:
fixed